packages/notifications/src/Http/Livewire/Concerns/CanNotify.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Filament\Notifications\Http\Livewire\Concerns;
+
+use Filament\Notifications\Notification;
+
+trait CanNotify
+{
+    protected function notify(Notification $notification): void
```

Comment:
This needs to have the same signature as in the existing `CanNotify` trait, but you can add the `Notification | string $status` union type to the first parameter

Reply:
Fixed in 96380d19325096b3625822e7bda43d5114c86ba1.